modules/compliance-supported-profiles.adoc

```diff
@@ -43,6 +43,14 @@ The Compliance Operator provides the following compliance profiles:
 |link:https://www.nerc.com/pa/Stand/Pages/CIPStandards.aspx[North American Electric Reliability Corporation (NERC)]
 |0.1.44+
 
+|ocp4-pci-dss
+|link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI-DSS v3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4]
```

Comment:
```suggestion
|link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI-DSS 3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4]
```

Reply:
Or, if you prefer, spell out "version"

Reply:
Same issue as below - This title is part of the code and would require a code change to fix.

Reply:
hardcoded